packages/devtools_app/lib/src/inspector/flutter/inspector_tree_flutter.dart

```diff
@@ -211,6 +219,11 @@ class _InspectorTreeState extends State<InspectorTree>
     constraintDisplayController?.dispose();
```

Comment:
init the focusNode in initState and dispose in the dispose in the dispose method to handle the lifecycle better.

Reply:
you might also consider adding a https://api.flutter.dev/flutter/widgets/FocusScope-class.html widget to the tree but I'm minimally familiar with managing focus in Flutter.

Reply:
FocusScope sounds like what we want here.

Reply:
We are only building with tip of trunk flutter so you can use FocusTraversalGroup now.